code/modules/mob/living/carbon/alien/humanoid/life.dm

```diff
@@ -1,28 +1,3 @@
-/mob/living/carbon/alien/humanoid/Life(seconds, times_fired)
-	. = ..()
-	update_icons()
```

Comment:
Once this refactor is done, I'll probably move onto improving Xeno code---complete with new icons.

How xenos currently work for updating icons is horrifying.